content/monitors/faq/as-count-monitor-evaluation.md

```diff
@@ -7,30 +7,30 @@ kind: faq
 
 Currently monitors that have the `as_count` modifier use a separate evaluation path than other monitors. In certain use-cases this this can result in unexpected behavior. We intend to migrate all monitors with `as_count` to the same evaluation path as other monitors, but this doc explains the underlying issue and will guide you through the migration process.
 
-Let's call the current evaluation path for `as_count` monitors `current_eva_path` and the new one `new_eva_path`.  
+Let's call the current evaluation path for `as_count` monitors `current_eval_path` and the new one `new_eval_path`.  
```

Comment:
So we're keeping `current_eval_path` and not `old_eval_path` as Dmytro mentioned ?

Reply:
Hum, probably `current` is better because otherwise people will think that `new` is already in prod